javascript/src/gherkin-languages.json

```diff
@@ -417,6 +417,55 @@
       "Kada "
     ]
   },
+  "by": {
```

Comment:
Hello!
The Belarusian language code is `be`, not `by`. Language code is not always the same as country code.
https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=be

Reply:
wow. thank you @skarpesh. Fixed